plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx

```diff
@@ -0,0 +1,155 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GroupEntity } from '@backstage/catalog-model';
+import { ApiProvider, ApiRegistry } from '@backstage/core';
+import {
+  CatalogApi,
+  catalogApiRef,
+  EntityProvider,
+} from '@backstage/plugin-catalog-react';
+import { renderWithEffects, wrapInTestApp } from '@backstage/test-utils';
+import { queryByText } from '@testing-library/react';
+import React from 'react';
+import { OwnershipCard } from './OwnershipCard';
+
+describe('OwnershipCard', () => {
+  const userEntity: GroupEntity = {
+    apiVersion: 'backstage.io/v1alpha1',
+    kind: 'Group',
+    metadata: {
+      name: 'my-team',
+    },
+    spec: {
+      type: 'team',
+      children: [],
+    },
+    relations: [
+      {
+        type: 'memberOf',
+        target: {
+          kind: 'group',
+          name: 'ExampleGroup',
+          namespace: 'default',
+        },
+      },
+    ],
+  };
+
+  it('displays entity counts', async () => {
+    const catalogApi: jest.Mocked<CatalogApi> = {
+      getEntities: jest.fn(),
+    } as any;
+
+    catalogApi.getEntities.mockResolvedValue({
+      items: [
+        {
+          kind: 'API',
+          metadata: {
+            name: 'my-api',
+          },
+          spec: {
+            type: 'openapi',
+          },
+          relations: [
+            {
+              type: 'ownedBy',
+              target: {
+                name: 'my-team',
+                namespace: 'default',
+                kind: 'Group',
+              },
+            },
+          ],
+        },
+        {
+          kind: 'Component',
+          metadata: {
+            name: 'my-service',
+          },
+          spec: {
+            type: 'service',
+          },
+          relations: [
+            {
+              type: 'ownedBy',
+              target: {
+                name: 'my-team',
+                namespace: 'default',
+                kind: 'Group',
+              },
+            },
+          ],
+        },
+        {
+          kind: 'Component',
+          metadata: {
+            name: 'my-library',
+            namespace: 'other-namespace',
+          },
+          spec: {
+            type: 'library',
+          },
+          relations: [
+            {
+              type: 'ownedBy',
+              target: {
+                name: 'my-team',
+                namespace: 'default',
+                kind: 'Group',
+              },
+            },
+          ],
+        },
+      ] as any,
+    });
+
+    const { getByText } = await renderWithEffects(
```

Comment:
Could just use `renderInTestApp`, same thing

Reply:
Oh, I actually thought that I used it here, copy and paste 😉